go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/containers/buildah v1.11.2
 	github.com/containers/conmon v2.0.0+incompatible
 	github.com/containers/image v3.0.2+incompatible
-	github.com/containers/libpod v1.5.1
+	github.com/containers/libpod v1.6.1
```

Comment:
why the libpod bump?

Reply:
For the annotation to be available, which has been introduced in 1.6.0.

Reply:
right silly me :upside_down_face: